flake.nix

```diff
@@ -63,6 +63,11 @@
       });
 
       checks.x86_64-linux.tarball = jobs.tarball;
+      devShell = {
+        x86_64-linux = import ./shell.nix { system = "x86_64-linux"; };
+        x86_64-darwin = import ./shell.nix { system = "x86_64-darwin"; };
+        aarch64-linux = import ./shell.nix { system = "aarch64-linux"; };
```

Comment:
Do we really have to manually list every platform here?  What happens to somebody who isn't on one of those three platforms?  (aarch64-darwin and powerpc64le-linux come to mind.)

Reply:
When writing I had a bad stomach feeling, already. :wink:

`numtide/flake-utils` has a `defaultSystem`. Do we have something similar go enumerate on the "blessed" set of systems as flake spec requires?

Maybe we also could just add those two to suport 'most common' platforms? What do you think?

Reply:
Somebody on a different platfrom currenrly still can try `nix-shell` directly using `shell.nix` circumventing `flake.nix`.